tools/bundletool/process_and_sign.sh.template

```diff
@@ -30,6 +30,40 @@ if [[ -n "$POST_PROCESSOR" ]]; then
   "$POST_PROCESSOR" "$WORK_DIR"
 fi
 
+# Strip out any unnecessary slices from embedded dynamic frameworks to save space
+for i in "$WORK_DIR"/*.app; do
+    # Lets figure out what slices the app has
+    app_name=$(basename $i .app)
+    slices=$(xcrun lipo -info $i/$app_name | cut -d: -f3)
```

Comment:
I think `app_slices` would be more clear here since there's also `framework_slices` below